mlflow-tests/tests/actions/artifact_actions.py

```
@@ -266,20 +392,38 @@ def action_wait_for_mlflowconfig_active(test_context: TestContext) -> None:
+        exp_id = None
         try:
-            exp_id = mlflow.create_experiment(f"mlflowconfig-probe-{i}")
-            test_context.add_experiment_for_cleanup(exp_id, test_context.active_workspace)
+            exp_id = mlflow.create_experiment(f"mlflowconfig-probe-{uuid.uuid4().hex}")
             location = mlflow.get_experiment(exp_id).artifact_location
 
-            if location and location.startswith("s3://") and path in location:
+            if location and location.startswith(expected_prefix):
                 logger.info(f"MLflowConfig active after {i + 1} attempt(s)")
                 return
         except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)